eo-runtime/src/main/java/EOorg/EOeolang/AtMemoized.java

```diff
@@ -81,6 +89,23 @@ public Phi get() {
 
     @Override
     public void put(final Phi phi) {
+        if (this.object != null) {
+            final byte[] enclosure = new Dataized(
+                new PhMethod(this.object, "as-bytes")
+            ).take(byte[].class);
+            final byte[] provided = new Dataized(
+                new PhMethod(phi, "as-bytes")
+            ).take(byte[].class);
+            if (provided.length > enclosure.length) {
```

Comment:
@Graur what would a result if I try to write `int` firstly, and `bool` secondary. I think there will be no error, but it seems to be a wrong logic of `memory`. Maybe at this step just prohibit to write objects with various amount of bytes? Then it would be something like: `if (provided.length != enclosure.length)`